### ISSUE TITLE
Dockerfile integration test

### ISSUE THREAD

**Issue by dmzaytsev** [yegor256/rultor#909]
right now Rultor doesn't test Dockerfile during merge with master branch of the project.
I believe it has to be. A bug in Dockerfile can break all Rultor's builds.
http://www.rultor.com/t/4577-124057172
http://www.rultor.com/t/4599-124002168

**Comment by yegor256** [comment]
@dmzaytsev that's what we see there:

```
sudo gem install --no-ri --no-rdoc pdd Building native extensions. This could take a while... Building native extensions. This could take a while... ERROR:	Error installing pdd: ERROR: Failed to build gem native extension.  /usr/bin/ruby1.9.1 extconf.rb checking for magic_open() in -lmagic... no *** ERROR: missing required library to compile this module *** extconf.rb failed *** Could not create Makefile due to some reason, probably lack of necessary libraries and/or headers. Check the mkmf.log file for more details. You may need configuration options.  Provided configuration options: --with-opt-dir --without-opt-dir --with-opt-include --without-opt-include=${opt-dir}/include --with-opt-lib --without-opt-lib=${opt-dir}/lib --with-make-prog --without-make-prog --srcdir=. --curdir --ruby=/usr/bin/ruby1.9.1 --with-magic-dir --without-magic-dir --with-magic-include --without-magic-include=${magic-dir}/include --with-magic-lib --without-magic-lib=${magic-dir}/lib --with-magiclib --without-magiclib   Gem files will remain installed in /var/lib/gems/1.9.1/gems/ruby-filemagic-0.6.1 for inspection. Results logged to /var/lib/gems/1.9.1/gems/ruby-filemagic-0.6.1/ext/filemagic/gem_make.out
```

**Comment by yegor256** [comment]
@dmzaytsev fixed in b644c20 in about 20 minutes a new Docker image will ready in the hub: https://registry.hub.docker.com/u/yegor256/rultor/build_id/77292/code/bf7mma6zyedt63a2rldqnfv/
